src/platform_impl/linux/wayland/window/mod.rs

```diff
@@ -620,6 +620,11 @@ impl Window {
         self.event_loop_awakener.ping();
     }
 
+    #[inline]
+    pub fn set_theme(&self, theme: Theme) {
```

Comment:
Could you remove `set_csd_theme` and related APIs? Also add a breaking change that it was removed in favor of unified `set_theme` API?

I think other backends might have similar APIs in place(X11's gtk thing is special though), might worth unifying as well.

Reply:
I took a stab at it but I was not sure if mapping `Theme::Dark` to `dark` and `Theme::Light` to `light` is correct, so let me know if they are wrong.